src/components/FormMap/index.tsx

```diff
@@ -0,0 +1,46 @@
+import './styles.scss';
+import { FloatingButton } from '@entur/button';
+import { UndoIcon } from '@entur/icons';
+import DefaultMapContainer from './DefaultMapContainer';
+import React from 'react';
+import { useConfig } from '../../config/ConfigContext';
+import FintrafficMapProvider from '../../ext/Fintraffic/CustomMapProvider';
+
+type Props = {
+  undo?: () => void;
+  children: React.ReactElement;
+};
+
+const FormMap = (props: Props) => {
+  const { extPath, sandboxFeatures } = useConfig();
+  return (
+    <div className="map-container eds-contrast">
+      {sandboxFeatures?.Fintraffic ? (
+        <FintrafficMapProvider>{props.children}</FintrafficMapProvider>
+      ) : (
+        <DefaultMapContainer>{props.children}</DefaultMapContainer>
+      )}
+      {/*<SandboxFeature
```

Comment:
We need to have a chat about this, there were problems in the map interactions once wrapping it with Sandbox

Reply:
Should be fixed by d314a2e5c44413a027208cd5ae71d89565c17869